spec/facter/resolvers/linux/processors_lscpu_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+describe Facter::Resolvers::Linux::Processors do
```

Comment:
This should be `Facter::Resolvers::Linux::Lscpu`. The filename should also be changed to `lscpu_spec.rb`.